### PR TITLE
Raidboss: t7 Change regex to netRegex 

### PR DESCRIPTION
**Pull request by VanSlanzar**
Change regex to netRegex for cursed and shriek voice.

### CODE REVIEW

File: ui/raidboss/data/02-arr/raid/t7.js [hunk 1]
```diff
@@ -82,12 +82,12 @@
     },
     {
       id: 'T7 Voice',
-      regex: Regexes.gainsEffect({ effect: 'Cursed Voice' }),
-      regexDe: Regexes.gainsEffect({ effect: 'Stimme Der Verwünschung' }),
-      regexFr: Regexes.gainsEffect({ effect: 'Voix Du Maléfice' }),
-      regexJa: Regexes.gainsEffect({ effect: '呪詛の声' }),
-      regexCn: Regexes.gainsEffect({ effect: '诅咒之声' }),
-      regexKo: Regexes.gainsEffect({ effect: '저주의 목소리' }),
+      netRegex: NetRegexes.gainsEffect({ effectId: '1C3', source: 'Melusine' }),
```
Comment (quisquous) [inline ui/raidboss/data/02-arr/raid/t7.js:85]: Why are you adding `source` here? From your log line pictures that you linked to me, it doesn't look like there's a source.

Comment (VanSlanzar) [reply ui/raidboss/data/02-arr/raid/t7.js:85]: Ah, https://github.com/quisquous/cactbot/blob/master/docs/LogGuide.md#1a-networkbuff
`1A:[ObjectId]:[Target Name] gains the effect of [Status] from [Source Name] for [Float_Value] Seconds`.

I believed in an oversight on my part, if it is useless, I correct that.

Comment (quisquous) [reply ui/raidboss/data/02-arr/raid/t7.js]: It's not that it's useless--it's that it's incorrect.  Looking at the log lines you sent me, there is no `source` from those gains effect lines.  It is empty.

Comment (VanSlanzar) [reply ui/raidboss/data/02-arr/raid/t7.js]: Yes, i understand. It's corrected.

File: ui/raidboss/data/02-arr/raid/t7.js [hunk 2]
```diff
@@ -82,12 +82,12 @@
     },
     {
       id: 'T7 Voice',
```
Comment (panicstevenson) [inline ui/raidboss/data/02-arr/raid/t7.js:84]: Can this be T7 Cursed Voice? I know it's kind of pedantic, but we lose some context by moving to the effectId

Comment (quisquous) [reply ui/raidboss/data/02-arr/raid/t7.js:84]: And `T7 Cursed Shriek` elsewhere too!

Comment (VanSlanzar) [reply ui/raidboss/data/02-arr/raid/t7.js:84]: This was done to resolve this issue. #1425 
As I did not understand the meaning of @panicstevenson  what should I do? @quisquous

Comment (JLGarber) [reply ui/raidboss/data/02-arr/raid/t7.js:84]: What they are saying is that "voice" as a trigger ID in English does not really explain what is going on with this trigger. Instead, it should be:

```
 {
      id: 'T7 Cursed Voice',
      // trigger body etc
  }
```

This way, we know what the mechanic is without having to look for clues or looking it up on XIVAPI.

(This is not your fault, the trigger was named this a very long time ago when we didn't have as much standardization for things. However, now that we are using bare effect IDs, we do need to have the name for it *somewhere* in the trigger.)

**Comment by VanSlanzar** [comment]
Is this better? thanks to @JLGarber for his explanations, I understand better.

**Comment by quisquous** [comment]
Thanks, perfect!

**Pull request merged**